tests/Services/ElementTabProviderTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace DNADesign\Elemental\Tests\Services;
+
+use SilverStripe\Dev\SapphireTest;
+
+class ElementTabProviderTest extends SapphireTest
+{
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->markTestIncomplete('Does not yet test anything, remains to be implemented');
+    }
```

Comment:
I'd say remove this rather than adding incomplete test classes

Reply:
I've implemented this as I noticed it was not tested - more as a beginning and something that will constantly be of note until it becomes a thing. I should raise an issue for it (it's not fleshed out as it is not the focus of this work)

Reply:
https://github.com/dnadesign/silverstripe-elemental/issues/431